### PR TITLE
Clarify use of ksql.service.id for scaleout

### PR DESCRIPTION
**Pull request by rmoff**

### CODE REVIEW

File: docs/installation/server-config/config-reference.rst [hunk 1]
```diff
@@ -127,7 +127,10 @@ The Schema Registry URL path to connect KSQL to.
 ksql.service.id
 ---------------
 
-The service ID of the KSQL server. By default, the service ID of KSQL servers is ``default_``. The service ID is also used as
+The service ID of the KSQL server. This is used to define the KSQL cluster membership of a KSQL server instance. If multiple KSQL
```
[the remark this answers is not in the document]
Comment (miguno) [reply docs/installation/server-config/config-reference.rst]: Yes, this is important.  It's the combo of `ksql.server.id` and `bootstrap.servers` that uniquely defines a KSQL cluster.

[the remark this answers is not in the document]
Comment (rmoff) [reply docs/installation/server-config/config-reference.rst]: @miguno I've updated the text - please review. Thanks.

**Comment by rmoff** [comment]
@miguno I've updated the text - please review. Thanks.

File: docs/installation/server-config/config-reference.rst (hunk 1 above)
[the remark this answers is not in the document]
Comment (miguno) [reply docs/installation/server-config/config-reference.rst]: Only one addendum, feel free to merge after updating: I'd clarify to "they will form a KSQL cluster" (not just "they will form a cluster") or similar to be extra-clear on Kafka cluster vs. KSQL cluster.  It's fairly common for some users to believe that KSQL runs inside the Kafka cluster...

**Pull request merged**